src/core/toga/window.py

```diff
@@ -178,6 +178,10 @@ def full_screen(self, is_full_screen):
         self._is_full_screen = is_full_screen
         self._impl.set_full_screen(is_full_screen)
 
+    @property
+    def dpi(self):
+        return self._impl.dpi
```

Comment:
This also shouldn't be needed; we don't need/want to expose a public API for DPI at the interface layer; it's status will be a reliable impl-level interface